### DISCUSSION TITLE
Question: is there a way to only compile and export the PCK file separately?

### DISCUSSION THREAD

**Discussion by antv199** [abarichello/godot-ci#58] [Q&A]
I'm looking to use the CI file for automated compiling for Gotm.io, which only requires a pck file. Can this actually be done or is this a feature that the repo wasn't meant for?

**Comment by Calinou** [comment]
You can modify the [GitHub Actions workflow](https://github.com/abarichello/godot-ci/blob/master/.github/workflows/godot-ci.yml) to call `--export-pack <preset> <path>` instead of `--export <preset> <path>`.
